audiophiler/templates/base.html

```diff
@@ -44,6 +44,33 @@
                             My Files
                         </a>
                     </li>
+                    {% if file.author == auth_dict["uid"] or is_rtp or is_eboard %}
+                    <li class="dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
+                            <span class="glyphicon glyphicon-music"></span> Harold
+                            <span class="caret"></span>
+                        </a>
+                        <ul class="dropdown-menu">
+                            <li>
+                                <a href="/admin">
+                                    <span class="glyphicon glyphicon-user"></span> Admin
+                                </a>
+                            </li>
+                            <li>
+                                <a href="/selected">
+                                    <span class="glyphicon glyphicon-tasks"></span> Selected Harolds
+                                </a>
+                            </li>
+                        </ul>
+                    </li>
+                    {% endif %}
+                    {% if (file.author != auth_dict["uid"]) or (not is_rtp) or (not is_eboard) %}
```

Comment:
Why not make this an else?

Reply:
First time I've ever coded in flask, didn't know the syntax